app/journey/UniformJourney.scala

```diff
@@ -53,30 +56,44 @@ object UniformJourney {
   val emailAddressRegex = """(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])"""
 
   // $COVERAGE-OFF$
-  def ctTaxJourney[F[_] : cats.Monad](interpreter: Language[F, TellTypes, AskTypes]): F[Cr03Submission] = {
+  def ctTaxJourney[F[_] : cats.Monad](interpreter: Language[F, TellTypes, AskTypes])(cr01Feature: Boolean): F[Cr03Submission] = {
     import interpreter._
 
     for {
+      reasonReport <- ask[ReasonReportType]("what-is-the-reason-for-the-report") when cr01Feature
```

Comment:
I don't like this feature flag. I know we need it, but only for limited time. Should be removed as soon as possible.

Reply:
Yep! We need to keep that in mind! I guess once it s validated in prod we can remove it 👍